apidoc/Titanium/Geolocation/Geolocation.yml

```diff
@@ -1254,22 +1260,29 @@ properties:
     platforms: [android]
 
   - name: postalCode
-    summary: Postal code. On iOS, use `zipcode`.
+    summary: Postal code.
+    deprecated:
+        since: "7.0.0"
+        notes: Use the `zipcode` property for better parity instead.
     type: String
     platforms: [android]
 
   - name: zipcode
-    summary: Postal code. On Android, use `postalCode`.
+    summary: Postal code.
     type: String
-    platforms: [iphone, ipad]
+    since: { iphone: "0.9.0", ipad: "0.9.0", android: "7.0.0" }
+    platforms: [iphone, ipad, android]
 
   - name: country
     summary: Country name.
     type: String
     platforms: [android, iphone, ipad]
 
   - name: countryCode
-    summary: Country code. On iOS, use `country_code`.
+    summary: Country code.
+    deprecated:
+        since: "7.0.0"
+        notes: Use the `country_code` property for better parity instead.
```

Comment:
Why point them at "country_code"? While it may be available on iOS/Windows, it's name doesn't match the naming scheme. I would think we'd prefer "countryCode".